core/src/test/java/com/linecorp/armeria/client/retry/BackoffTest.java

```diff
@@ -62,11 +60,22 @@ public void exponential() throws Exception {
 
     @Test
     public void withJitter() throws Exception {
-        Random random = new Random(1);
-        Backoff backoff = Backoff.fixed(100).withJitter(-50, 50, () -> random);
-        assertThat(backoff.nextDelayMillis(1)).isEqualTo(120);
-        assertThat(backoff.nextDelayMillis(2)).isEqualTo(139);
-        assertThat(backoff.nextDelayMillis(3)).isEqualTo(94);
+        int maxAttempts = 200;
+        Backoff backoff = Backoff.fixed(1000).withJitter(0.3).withMaxAttempts(maxAttempts);
+
+        for (int i = 1; i < maxAttempts; i++) {
+            assertThat(backoff.nextDelayMillis(i)).isBetween(700L, 1300L);
```

Comment:
Why change the style of the test from a seeded Random and fixed assertions? This doesn't seem to be a good test anymore, e.g., it's possible for a bug to cause no jitter to ever be added, so the value returned is always 1000 which still passes this test despite being broken.